resources/ar.ini

```diff
@@ -0,0 +1,45 @@
+; Updated time : 26th 07 2017
+; general
+divider = "========================================"
+spacer = " -=+=- {%0} -=+=- "
+error = "حصل خطأ ما"
+noperm = "لا تملك الصلاحيات الكافية لتفعيل هذا الأمر"
+runingame = "الرجاء تفعيل هذا الأمر داخل اللعبة !"
+commands = "الأوامر"
+; user interfaces
+ui.confirmation = "تأكيد"
+ui.confirmation.yes = "نعم"
+ui.confirmation.no = "لا"
+; ui brush
+ui.brush.title = "قائمة الفرشاة"
+; ui brush select
+ui.brush.select.title = "اختر نوع الفرشاة"
+ui.brush.select.type.sphere = "كرة"
+ui.brush.select.type.cylinder = "اسطوانة"
+ui.brush.select.type.cuboid = "مكعب"
+ui.brush.select.type.clipboard = "حافظة"
+; ui brush settings
+ui.brush.settings.title = "{%0} اعدادات الفرشاة"
+; ui brush options
+ui.brush.options.blocks = "كتل"
+ui.brush.options.blocks.placeholder = "مثال :  1:1,2,tnt,log:12"
```

Comment:
In `eng.ini`
`ui.brush.options.blocks.placeholder = "Example: 1:1,2,tnt,log:12"`
In `ara.ini`
`ui.brush.options.blocks.placeholder = "مثال :  1:1,2,tnt,log:12"`

Is it possible to change `مثال ` to the beginning? Else, just put it at the end instead; either or, it just shouldn't be inside the ID list.

Reply:
it's in the beginning like this, it shows like this because it's an RTL language displayed in LTR
just copy `مثال : 1:1,2,tnt,log:12` and paste it in google search it would show up correctly

<img width="1440" alt="screen shot 2018-10-29 at 10 16 58 am" src="https://user-images.githubusercontent.com/13835834/47637267-d36a3b80-db63-11e8-8051-26b7c80773b7.png">

Reply:
Ah, I see. In that case, it looks fine to merge. Thank you!